lib/cfg.c

```diff
@@ -329,6 +329,24 @@ cfg_new(gint version)
   return self;
 }
 
+GlobalConfig *
+cfg_load_config_snippet(const gchar *fname)
+{
+  GlobalConfig *cfg = cfg_new(0);
```

Comment:
About the second sentence, I absolutely agree with you. The sad fact is that we _already use it_ in this way internally. The bug report was to prevent loading modules in this case. 

I'll pass the config version here.